src/data/reducers/page.js

```diff
@@ -7,6 +7,7 @@ import {
 
 const page = (state = {
   page: {},
+  is_preview: false,
```

Comment:
Don't need this because default-prop type is false.

Reply:
fixed